tests/integration/Cron/Workers/BeamerTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace MailPoet\Test\Cron\Workers;
+
+use Carbon\Carbon;
+use Codeception\Stub;
+use MailPoet\Cron\Workers\Beamer;
+use MailPoet\Settings\SettingsController;
+use MailPoet\WP\Functions as WPFunctions;
+
+class BeamerTest extends \MailPoetTest {
+  function testItSetsLastAnnouncementDate() {
```

Comment:
Should there be more tests? You only test the happy path, but should we add a test for:
- 404 response from the Beamer API
- 200 response with an empty body
- incomplete JSON that fails to decode
- valid JSON but an unexpected structure